clients/src/main/java/org/apache/kafka/clients/consumer/internals/events/ApplicationEventProcessor.java

```diff
@@ -203,7 +203,9 @@ public void process(ApplicationEvent event) {
 
     private void process(final PollEvent event) {
         if (requestManagers.commitRequestManager.isPresent()) {
-            requestManagers.commitRequestManager.ifPresent(m -> m.updateAutoCommitTimer(event.pollTimeMs()));
+            CommitRequestManager commitRequestManager = requestManagers.commitRequestManager.get();
+            commitRequestManager.updateAutoCommitTimer(event.pollTimeMs());
+            commitRequestManager.allConsumed(event.allConsumed());
```

Comment:
Can we consolidate the auto-commit timer and the offsets map in a single method call that's more clearly named? Are there other places that call `updateAutoCommitTimer()` other than this path?